styles/xrpl.scss

```diff
@@ -53,3 +53,19 @@ $font-family-sans-serif: -apple-system, system-ui, 'Roboto', sans-serif;
 @import "_pages.scss";
 @import "_rpc-tool.scss";
 @import "_blog.scss";
+
+// Option to only change theme on user system settings.  No toggle.
+
+//Mixin to conditionaly include other themes
+// @import "light/_mixins.scss";
+
+// Default to dark theme with light being alt
+// $color-scheme-alt: light;
+
+// @include prefers-color-scheme($color-scheme-alt) {
+//     @import "light/_light-theme.scss";
+// }
```

Comment:
This was/is for a different way to handle the theme changes.  
If we went this direction it would load and respect only the users system pref.   Basically would load only if user had a light pref set.  But not able to handle user toggle on page.  As the content wouldn't be loaded.  Media query for css.  

Left in thinking it could be discussed as a potential option if you didn't need or want user to select manually.   
Looks like Github for example has a setting in your prefs for overriding the system settings and will add separate themes on multiple conditions.   
Unsure how far down that rabbit hole we go.